plugins/woocommerce-admin/client/subscriptions/index.tsx

```diff
@@ -0,0 +1,344 @@
+	const { installAndActivatePlugins } = useDispatch( PLUGINS_STORE_NAME );
+
+	return (
+		<Button
+			disabled={ isGettingStarted }
+			isBusy={ isGettingStarted }
+			isPrimary
+			onClick={ () => {
+				setIsGettingStarted( true );
+				recordEvent(
+					'wccore_subscriptions_empty_state_get_started_click',
+					{}
+				);
+				installAndActivatePlugins( [ 'woocommerce-payments' ] )
+					.then( () => {
+						window.location.href =
+							experimentAssignment === Treatment.A
+								? newSubscriptionProductUrl
+								: onboardingUrl;
+					} )
+					.catch( () => {
+						recordEvent(
+							'wccore_subscriptions_empty_state_get_started_error'
+						);
+						setIsGettingStarted( false );
+						setHasError( true );
+					} );
+			} }
+		>
+			{ __( 'Get started', 'woocommerce' ) }
+		</Button>
+	);
+};
+
+const StepNumber: React.FC = ( { children } ) => (
+	<span className="wcpay-empty-subscriptions-page-step-number">
+		{ children }
+	</span>
+);
+
+const TermsOfService = () => (
+	<span className="wcpay-empty-subscriptions-page-terms-of-service">
+		{ createInterpolateElement(
+			__(
+				'By clicking “Get started”, the WooCommerce Payments plugin will be installed and you agree to the <a>Terms of Service</a>',
+				'woocommerce'
+			),
+			{
+				a: (
+					// eslint-disable-next-line jsx-a11y/anchor-has-content
+					<a
+						href="https://wordpress.com/tos/"
+						target="_blank"
+						rel="noreferrer"
+					/>
+				),
+			}
+		) }
+	</span>
+);
+
+type MainContentProps = {
+	setHasError: setHasErrorFunction;
+};
+
+const MainContent: React.FC< MainContentProps > = ( { setHasError } ) => {
+	return (
+		<>
+			<h2>
+				{ __( 'Start selling subscriptions today', 'woocommerce' ) }
+			</h2>
+			<p>
+				{ __(
+					'With WooCommerce Payments, you can sell subscriptions with no setup costs or monthly fees. Create subscription products, track recurring revenue, and manage subscribers directly from your store’s dashboard.',
+					'woocommerce'
+				) }
+				<br />
+				<a
+					href="https://woocommerce.com/document/payments/subscriptions/"
+					target="_blank"
+					rel="noreferrer"
+				>
+					{ __( 'Learn more', 'woocommerce' ) }
+				</a>
+			</p>
+
+			<h3>{ __( 'Accepted payment methods', 'woocommerce' ) }</h3>
+
+			<div className="wcpay-empty-subscriptions-page-payment-methods">
+				<Visa />
+				<MasterCard />
+				<Amex />
+				<DinersClub />
+				<Discover />
+				<UnionPay />
+				<JCB />
+			</div>
+
+			<hr />
+
+			<p className="subscriptions__action">
+				<TermsOfService />
+			</p>
+
+			<div className="wcpay-empty-subscriptions__button_container">
+				<GetStartedButton setHasError={ setHasError } />
+				<NoThanksButton />
+			</div>
+		</>
+	);
+};
+
+const OnboardingSteps = () => (
+	<>
+		<h2>
+			{ __(
+				'You’re only steps away from selling subscriptions',
+				'woocommerce'
+			) }
+		</h2>
+		<div className="subscriptions-page-onboarding-steps">
```

Comment:
I don't know if we have a strict standard around this, but most of our components follow a BEM pattern for classes.

`subscriptions-page__onboarding-steps`

Reply:
Thanks @joshuatf. I've fixed this and the other class names in [9166928](https://github.com/woocommerce/woocommerce/pull/32957/commits/91669281ec5aeee6d26019be20c7b413ccd8e2ee)